app/services/foreman_discovery/import_hooks/discovery_attribute.rb

```diff
@@ -27,6 +32,14 @@ def import_from_facts
           disk_count: disk_count,
           disks_size: disks_size
         }
+      rescue Exception => e
+        ::Foreman::Logging.exception("Unable to parse attributes from facts", e)
```

Comment:
Do you think it's a good idea to swallow any errors here? Users will never report bugs and we'll never fix them if we just log them.

Reply:
Yes this is intentional in this case, if there is a parsing bug then we at least end up with a valid discovered host and the error is logged properly.

Reply:
However if you don't like this, I can remove. It was added as an "extra idea"...

Reply:
I guess we should think about having a public sentry instance to collect all exceptions so we can do something about them. I'll leave it to you to decide.

Reply:
I am going to drop this from the patch, it is really not relevant and I don't want to introduce another regression.